examples/sellar_state_MDF_optimize.py

```diff
@@ -144,7 +141,7 @@ def __init__(self):
         self.connect('d2.y2', 'state_eq.y2_actual')
 
         self.add('obj_cmp', ExecComp('obj = x**2 + z[1] + y1 + exp(-y2)',
-                                     z=np.array([0.0, 0.0]), x=0.0, d1=0.0, d2=0.0),
+                                     z=np.array([0.0, 0.0]), x=0.0, y1=0.0, y2=0.0),
```

Comment:
If the initial value is 0.0, you really don't have to set it in the args because that's the default value it'll get.